hack/release-operatorhub

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# This script generates bundle directories to be imported into OperatorHub.io.s
+
+set -euo pipefail
+
+output_dir=''
+
+make_output_dir () {
+  if ! [ -d "$output_dir" ]; then
+    mkdir "$output_dir"
+  else
+    rm -fr "${output_dir:?}"/*
+  fi
+}
+
+created_at=$(date +'%Y-%m-%d %H:%M:%S')
+readonly created_at
+container_image=''
+csv_path=''
+
+replace_csv () {
+  yq -i "
+    .metadata.annotations.containerImage = \"$container_image\" |
```

Comment:
How about we just pass the image name as an argument to this function instead of using a global variable?
If we want to keep a meaningful name in the function we can always use `container_image=$1`. 

WDYT?